From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Wed, 18 May 2022 15:49:05 +0800
Subject: gdk/: Rename gdk-pub-headers.h to gdk-sources.inc

We want to make the sources listings for libgdk shared as well.

Origin: upstream, 3.24.35, commit:f324c650a717d2d0a12faafa7a8c2f2888ace72e
---
 gdk/Makefile.am             |  2 +-
 gdk/gdk-pub-headers.mak     | 44 --------------------------------------------
 gdk/gdk-sources.inc         | 44 ++++++++++++++++++++++++++++++++++++++++++++
 win32/create-lists-msvc.mak |  2 +-
 4 files changed, 46 insertions(+), 46 deletions(-)
 delete mode 100644 gdk/gdk-pub-headers.mak
 create mode 100644 gdk/gdk-sources.inc

diff --git a/gdk/Makefile.am b/gdk/Makefile.am
index 5eae5a1..619ca01 100644
--- a/gdk/Makefile.am
+++ b/gdk/Makefile.am
@@ -1,6 +1,6 @@
 ## Makefile.am for gtk+/gdk
 include $(top_srcdir)/Makefile.decl
-include gdk-pub-headers.mak
+include gdk-sources.inc
 
 -include $(INTROSPECTION_MAKEFILE)
 INTROSPECTION_GIRS =
diff --git a/gdk/gdk-pub-headers.mak b/gdk/gdk-pub-headers.mak
deleted file mode 100644
index e84c5d0..0000000
--- a/gdk/gdk-pub-headers.mak
+++ /dev/null
@@ -1,44 +0,0 @@
-#
-# GDK header files for public installation (non-generated)
-#
-#
-gdk_public_h_sources = 				\
-	gdk.h					\
-	gdk-autocleanup.h			\
-	gdkapplaunchcontext.h			\
-	gdkcairo.h				\
-	gdkcursor.h				\
-	gdkdevice.h				\
-	gdkdevicepad.h				\
-	gdkdevicetool.h				\
-	gdkdevicemanager.h			\
-	gdkdisplay.h				\
-	gdkdisplaymanager.h			\
-	gdkdnd.h				\
-	gdkdrawingcontext.h			\
-	gdkevents.h				\
-	gdkframetimings.h			\
-	gdkglcontext.h				\
-	gdkkeys.h				\
-	gdkkeysyms.h				\
-	gdkkeysyms-compat.h			\
-	gdkmain.h				\
-	gdkmonitor.h				\
-	gdkpango.h				\
-	gdkframeclock.h				\
-	gdkpixbuf.h				\
-	gdkprivate.h				\
-	gdkproperty.h				\
-	gdkrectangle.h				\
-	gdkrgba.h				\
-	gdkscreen.h				\
-	gdkseat.h				\
-	gdkselection.h				\
-	gdktestutils.h				\
-	gdkthreads.h				\
-	gdktypes.h				\
-	gdkvisual.h				\
-	gdkwindow.h
-
-gdk_deprecated_h_sources =				\
-	deprecated/gdkcolor.h
diff --git a/gdk/gdk-sources.inc b/gdk/gdk-sources.inc
new file mode 100644
index 0000000..e84c5d0
--- /dev/null
+++ b/gdk/gdk-sources.inc
@@ -0,0 +1,44 @@
+#
+# GDK header files for public installation (non-generated)
+#
+#
+gdk_public_h_sources = 				\
+	gdk.h					\
+	gdk-autocleanup.h			\
+	gdkapplaunchcontext.h			\
+	gdkcairo.h				\
+	gdkcursor.h				\
+	gdkdevice.h				\
+	gdkdevicepad.h				\
+	gdkdevicetool.h				\
+	gdkdevicemanager.h			\
+	gdkdisplay.h				\
+	gdkdisplaymanager.h			\
+	gdkdnd.h				\
+	gdkdrawingcontext.h			\
+	gdkevents.h				\
+	gdkframetimings.h			\
+	gdkglcontext.h				\
+	gdkkeys.h				\
+	gdkkeysyms.h				\
+	gdkkeysyms-compat.h			\
+	gdkmain.h				\
+	gdkmonitor.h				\
+	gdkpango.h				\
+	gdkframeclock.h				\
+	gdkpixbuf.h				\
+	gdkprivate.h				\
+	gdkproperty.h				\
+	gdkrectangle.h				\
+	gdkrgba.h				\
+	gdkscreen.h				\
+	gdkseat.h				\
+	gdkselection.h				\
+	gdktestutils.h				\
+	gdkthreads.h				\
+	gdktypes.h				\
+	gdkvisual.h				\
+	gdkwindow.h
+
+gdk_deprecated_h_sources =				\
+	deprecated/gdkcolor.h
diff --git a/win32/create-lists-msvc.mak b/win32/create-lists-msvc.mak
index 02eada5..0d99a00 100644
--- a/win32/create-lists-msvc.mak
+++ b/win32/create-lists-msvc.mak
@@ -34,7 +34,7 @@
 NULL=
 
 # For GDK enumeration sources
-!include ..\gdk\gdk-pub-headers.mak
+!include ..\gdk\gdk-sources.inc
 
 !if [call create-lists.bat header gdk_headers.mak GDK_PUBLIC_HEADERS]
 !endif
